### ISSUE TITLE
Chart information is not accessible using keyboard: A11y_.NET 9 template_.NET 9 template_Home_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26234]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG2.1.1; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #Keyboard; #A11yMAS;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Observe that Chart information is not accessible using keyboard.

### Actual Result:
Chart information is not accessible using keyboard.

### Expected Result:
Chart information should be accessible using the keyboard:

### User Impact:
keyboard users will not be able to get the chart data.

### Attachment:
https://github.com/user-attachments/assets/1be963b8-5d4c-48f1-87e4-d91a05b1f873

**Comment by jsuarezruiz** [comment]
Related PR https://github.com/dotnet/maui/pull/27058

**Comment by AnKushSingh05** [comment]
Issue is fixed, hence closing the bug

https://github.com/user-attachments/assets/3eb76e1f-cfe5-4131-8579-8bbd0d441ccf
